src/environment_react_native.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+var DeviceEventEmitter = require('react-native').DeviceEventEmitter;
+var WebRTC = require('react-native-webrtc');
```

Comment:
I'm not familiar with the react-native workflow, but I'm assuming that it automatically provides the `react-native` dependency. Is that the case?

What about the `react-native-webrtc` dependency? I'm guessing it doesn't get automatically provided by react-native, so is the assumption that the developer will have to manually install it alongside `sip.js`?